go.mod

```diff
@@ -1,19 +1,92 @@
 module github.com/openstack-k8s-operators/cinder-operator
 
-go 1.13
+go 1.18
 
 require (
-	github.com/blang/semver v3.5.1+incompatible
-	github.com/go-logr/logr v0.1.0
-	github.com/onsi/ginkgo v1.12.1
-	github.com/onsi/gomega v1.10.1
-	github.com/openshift/api v0.0.0-20200205133042-34f0ec8dab87
-	github.com/openstack-k8s-operators/keystone-operator v0.0.0-20201020115836-d8f73f3ff674
-	github.com/openstack-k8s-operators/lib-common v0.0.0-20201012132655-247b83b2fafa
-	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20200321030439-57b580e57e88
-	github.com/prometheus/common v0.7.0
-	k8s.io/api v0.18.6
-	k8s.io/apimachinery v0.18.6
-	k8s.io/client-go v0.18.6
-	sigs.k8s.io/controller-runtime v0.6.2
+	github.com/go-logr/logr v1.2.3
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.18.1
+	github.com/openshift/api v3.9.0+incompatible
+	github.com/openstack-k8s-operators/keystone-operator v0.0.0-20220708133335-266d701ad809
+	github.com/openstack-k8s-operators/lib-common v0.0.0-20220721205745-5fb0d48a064d
```

Comment:
this looks to be from before reorg

Reply:
Yeah, there is a bunch of stuff that needs to be changed now since this PR is a couple weeks old.  I'm ambivalent as to whether to integrate the new `lib-common` reorg here or wait for a follow-up PR.

Reply:
New `lib-common` is now used